tabled/Cargo.toml

```diff
@@ -239,6 +239,11 @@ name = "alphabet"
 path = "examples/alphabet.rs"
 required-features = ["std"]
 
+[[example]]
+name = "split"
+path = "examples/split.rs"
+required-features = []
```

Comment:
Add here `"macros"` to fix a CI.

Reply:
Thank you!
https://github.com/zhiburt/tabled/blob/f5c56a4b50a5e34a04035bc19cefb7e088b22a8f/tabled/Cargo.toml#L242-L243

I was thinking that this block of code meant that the macros feature was enabled by default, and was scratching my head why those checks were still failing 😅